string personality
string action
string move
---
bool success
